parts/linux/cloud-init/artifacts/cse_install.sh

```diff
@@ -201,6 +201,16 @@ installImg() {
     retrycmd_get_executable 120 5 $img_filepath "https://acs-mirror.azureedge.net/img/img-linux-amd64-v0.5.6" ls || exit $ERR_IMG_DOWNLOAD_TIMEOUT
 }
 
+extractKubeBinaries() {
+    KUBE_BINARY_URL=${KUBE_BINARY_URL:-"https://kubernetesartifacts.azureedge.net/kubernetes/v${KUBERNETES_VERSION}/binaries/kubernetes-node-linux-amd64.tar.gz"}
+    K8S_TGZ_TMP=$(echo ${KUBE_BINARY_URL} | cut -d "/" -f 5)
```

Comment:
use the ##*/ syntax, eg.  CNI_TGZ_TMP=${CNI_PLUGINS_URL##*/} 

This way it doesn't care how many / we have in the url

Reply:
also pls install one 1.19 to verify - it we don't have enough disk space we can remove it later.